[BUG]: Cannot download newer docker image after release of 1.29.1

After release of 1.29.1 started to get message to update docker image:
```
E       playwright._impl._api_types.Error: Executable doesn't exist at /ms-playwright/chromium-1033/chrome-linux/chrome
E       ╔═══════════════════════════════════════════════════════════════╗
E       ║ Looks like Playwright was just updated to 1.28.0.             ║
E       ║ Please update docker image as well.                           ║
E       ║ -  current: mcr.microsoft.com/playwright/python:v1.29.1-focal ║
E       ║ - required: mcr.microsoft.com/playwright/python:v1.28.0-focal ║
E       ║                                                               ║
E       ║ <3 Playwright Team                                            ║
E       ╚═══════════════════════════════════════════════════════════════╝
```

but looks like there is no image available:
```
docker pull mcr.microsoft.com/playwright/python:v1.29.1-focal                                          
Error response from daemon: manifest for mcr.microsoft.com/playwright/python:v1.29.1-focal not found: manifest unknown: manifest tagged by "v1.29.1-focal" is not found
```

--- a/packages/playwright-core/src/server/registry/dependencies.ts
+++ b/packages/playwright-core/src/server/registry/dependencies.ts
@@ -26,7 +26,7 @@ import { deps } from './nativeDeps';
 import { getPlaywrightVersion } from '../../common/userAgent';
 
 const BIN_DIRECTORY = path.join(__dirname, '..', '..', '..', 'bin');
-const packageJSON = require('../../../package.json');
+const languageBindingVersion = process.env.PW_CLI_DISPLAY_VERSION || require('../../../package.json').version;
 
 const dockerVersionFilePath = '/ms-playwright/.docker-info';
 export async function writeDockerVersion(dockerImageNameTemplate: string) {
@@ -38,8 +38,8 @@ export async function writeDockerVersion(dockerImageNameTemplate: string) {
 
 export function dockerVersion(dockerImageNameTemplate: string): { driverVersion: string, dockerImageName: string } {
   return {
-    driverVersion: packageJSON.version,
-    dockerImageName: dockerImageNameTemplate.replace('%version%', packageJSON.version),
+    driverVersion: languageBindingVersion,
+    dockerImageName: dockerImageNameTemplate.replace('%version%', languageBindingVersion),
   };
 }
 

